lib/permutations.rb

```diff
@@ -1,4 +1,25 @@
 
 def permutations?(string1, string2)
```

Comment:
This method isn't taking into account the numbers of each character.  So in this example "too" and "tot" would be considered permutations.